README.md

```diff
@@ -35,6 +35,7 @@ While chipStar 1.0 can already be used to run various large HPC applications suc
 
   * An OpenCL 2.0 or 3.0 driver with at least the following features supported:
     * Coarse-grained buffer Shared Virtual Memory
+    * OpenCL 2.0 atomics (CL\_DEVICE\_SVM\_ATOMICS)
```

Comment:
Please note that this section is for the _required_ extensions (for any input CUDA/HIP application). If the application doesn't use atomics, they are not required, right? We shall list them in optional extensions (similar to ballots, shuffles,...)

Reply:
I removed it, upon further reading `CL_DEVICE_SVM_ATOMICS` is not required.